main/boofcv-ip/src/main/java/boofcv/alg/color/ColorDiff.java

```diff
@@ -0,0 +1,240 @@
+package boofcv.alg.color;
+
+/**
+ * ColorDiff methods.
```

Comment:
Could you provide a slightly better explanation? E.g. Methods for computing the difference (or error) between two colors.